panel/src/components/Forms/Layouts/Layouts.vue

```diff
@@ -116,7 +116,7 @@ export default {
 			// replace all unique IDs for columns and blocks
 			copy = this.updateIds(copy);
```

Comment:
Could you add a comment above here that updateIds always returns an array? Do that our copy object now is an array. Maybe we should even rather use `const copies` for this line? Cause then your change makes a lot more sense when reading it. First I was very irritated why that change.

Reply:
I've improved the method and add a description https://github.com/getkirby/kirby/pull/5350/commits/12045057b6e92bf0788c3c00692b268592b62ad4